tests/bearlib/aspects/ClassTest.py

```diff
@@ -31,3 +33,10 @@ class docs:
                 example = 'Example'
 
         assert not SubAspect.docs.check_consistency()
+        assert issubaspect(SubAspect, RootAspect)
+        assert not issubaspect(Root, RootAspect)
+        assert not issubaspect(RootAspect, RootAspect)
```

Comment:
As mentioned in https://github.com/coala/coala/pull/3996#discussion_r108486591 this should be `True`.